oca_event_badge/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+#. create tags in partner form ('board', 'member', 'delegate')
```

Comment:
usually a list with one entry means that you do not need the list :wink: 

->  `To use this module, you need to create the tags in partner form ('board', 'member', 'delegate')`